### PR TITLE
Cover each class of ogci library with simple tests

### PR DESCRIPTION
**Pull request by alitet01**
## Cover each class of ogci library with simple tests
Use simple tests, one method of each class
- job: should point at groovy file `jobs/tests/all.groovy`
- tests:
    - test scripts should reside under `jobs/tests`
    - naming convention: `test${component}Simple.groovy`

### CODE REVIEW

File: jobs/tests/all.groovy
Comment (mvk) [inline jobs/tests/all.groovy:91]: @alitet01 maybe rename from here downwards:
```
groovies -> testFileNamesOutput
grooviesArr -> testFileNamesArr
groovyPath -> testFileName
```

**Review by mvk — CHANGES_REQUESTED** [review]
@alitet01 just rename as I request, and you can merge.

**Comment by alitet01** [comment]
надо переговорить, когда это удобно сделать?
Виктор

**Comment by alitet01** [comment]
waiting for approval

**Review by mvk — CHANGES_REQUESTED** [review]
@alitet01 please consolidate all the changes you've tested into this branch.

Thanks.

**Pull request closed without merging**
